titus_isolate/metrics/event_log.py

```diff
@@ -29,17 +29,30 @@ def get_event_msg(event):
 
 
 def __get_cpu_event(cpu: Cpu, usage: dict, workloads: dict):
+    cm = get_config_manager()
     return {
         "uuid": str(uuid.uuid4()),
         "payload": {
-            "instance": get_config_manager().get_str('EC2_INSTANCE_ID'),
+            "instance": cm.get_str('EC2_INSTANCE_ID'),
+            "region": cm.get_str('EC2_REGION'),
             "cpu": cpu.to_dict(),
             "cpu_usage": usage,
             "workloads": workloads
         }
     }
 
 
+def get_msg_ctx():
+    cm = get_config_manager()
+    return {
+        "uuid": str(uuid.uuid4()),
+        "payload": {
+            "instance": cm.get_str('EC2_INSTANCE_ID'),
```

Comment:
This is the instance and region where the solver service container happens to be running.  Are you sure you want that?

Reply:
Yes